packages/rolldown/tests/fixtures/tree-shake/annotations/_config.ts

```diff
@@ -3,6 +3,7 @@ import { defineTest } from 'rolldown-tests'
 import { expect } from 'vitest'
 
 export default defineTest({
+  skip: true,
```

Comment:
Need a tracking issue for this change.

Reply:
https://github.com/rolldown/rolldown/issues/3753